### PR TITLE
Websocket implementation

### PR DESCRIPTION
**Pull request by omangbaheti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,45 +1,44 @@
 # Overview
 
-Python script to stream data from vicon nexus to unity. See [vicon-nexus-unity-stream](https://github.com/ovi-lab/vicon-nexus-unity-stream) for the accompanying unity package.
+Sample project generated from Jace's Python Template.
```
Comment (ahmed-shariff) [inline README.md]: why did these change?

File: README.md [hunk 2]
```diff
@@ -1,45 +1,44 @@
...
 * Python 3.8+
 
-## Installation
+### Installation
 
 Install it directly into an activated virtual environment:
 
 ```text
-$ pip install vicon_nexus_unity_stream_py
+$ pip install ViNexPy
```
Comment (ahmed-shariff) [inline README.md]: can you rename everything back to vicon_nexus_unity_stream_py?

File: ViNexPy/__init__.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+__version__ = "0.2.1"
+
+# import logging
```
Comment (ahmed-shariff) [inline ViNexPy/__init__.py]: why are all of these commented out?

File: ViNexPy/_main.py [hunk 4]
```diff
@@ -241,14 +243,15 @@ def close(self):
         self._bar.__exit__()
         logger.configure()
 
+    
 
 def main(connection=None):
     client = get_client(connection)
+    pool = Pool(processes=8)
```
Comment (ahmed-shariff) [inline ViNexPy/_main.py]: is mp being used here? if not, remove em

**Comment by ahmed-shariff** [comment]
Fixed a few more things in both repos. If all good, I can merge em @omangbaheti

File: vicon_nexus_unity_stream_py/_main.py [hunk 5]
```diff
@@ -2,248 +2,100 @@
...
+    """Returns the vicon client instance."""
     if connection is None:
         connection = 'localhost:801'
     client = ViconDataStream.Client()
 
-    logger.info('Connecting...')
     while not client.IsConnected():
         client.Connect(connection)
-    logger.info('Connected to vicon data stream')
+    client.SetStreamMode(ViconDataStream.Client.StreamMode.EClientPullPreFetch)
```
Comment (omangbaheti) [inline vicon_nexus_unity_stream_py/_main.py:53]: On a sidenote: Do you want to experiment with Pull, Push and PullPrefetch to see if this is a factor for dropped frames in unity

Comment (omangbaheti) [reply vicon_nexus_unity_stream_py/_main.py:53]: but other than that
Saul Goodman 👍

Comment (ahmed-shariff) [reply vicon_nexus_unity_stream_py/_main.py:53]: good point. We'll do that in a sperate PR.

**Review by ahmed-shariff — APPROVED** [review]

**Pull request merged**